### PR TITLE
For overload, move properties defined in reference to its item

### PR DESCRIPTION
**Pull request by vicancy**

### CODE REVIEW

File: plugins/Microsoft.DocAsCode.Build.MemberLevelManagedReference/SplitClassPageToMemberLevel.cs [hunk 1]
```diff
@@ -498,6 +528,30 @@ private ImmutableArray<UidDefinition> CalculateUids(PageViewModel page, string f
...
+            var jarray = value as JArray;
+            if (jarray != null)
+            {
+                try
+                {
+                    return jarray.ToObject<List<string>>();
+                }
+                catch (Exception)
+                {
+                    Logger.LogWarning($"Unknown version metadata: {jarray.ToString()}");
```
Comment (superyyrrzz) [inline plugins/Microsoft.DocAsCode.Build.MemberLevelManagedReference/SplitClassPageToMemberLevel.cs]: >version [](start = 48, length = 7)

what's version?

**Comment by superyyrrzz** [comment]
:shipit:

File: test/Microsoft.DocAsCode.Build.ManagedReference.WithPlugins.Tests/SplitClassPageToMemberLevelTest.cs [hunk 2]
```diff
@@ -105,6 +105,15 @@ public void ProcessMrefWithTocShouldSucceed()
...
                 Assert.Equal(new List<string> { "net2", "net46" }, model.Platform);
+                Assert.Equal("<p sourcefile=\"TestData/mref/CatLibrary.Cat`2.yml\" sourcestartlinenumber=\"1\" sourceendlinenumber=\"1\">Overload summary</p>\n", model.Summary);
+                Assert.Equal("<p sourcefile=\"TestData/mref/CatLibrary.Cat`2.yml\" sourcestartlinenumber=\"1\" sourceendlinenumber=\"1\">Overload <em>remarks</em></p>\n", model.Remarks);
+                Assert.Equal(new List<string>
+                {
+                    "<p sourcefile=\"TestData/mref/CatLibrary.Cat`2.yml\" sourcestartlinenumber=\"1\" sourceendlinenumber=\"1\">Overload example 1</p>\n",
+                    "<p sourcefile=\"TestData/mref/CatLibrary.Cat`2.yml\" sourcestartlinenumber=\"1\" sourceendlinenumber=\"1\">Overload <strong>example 2</strong></p>\n"
+                }, model.Examples);
+                Assert.Equal("Not defined Property", model.Metadata["not-defined"]);
+                Assert.NotNull(model.Source);
```
Comment (hellosnow) [inline test/Microsoft.DocAsCode.Build.ManagedReference.WithPlugins.Tests/SplitClassPageToMemberLevelTest.cs]: >Source [](start = 37, length = 6)

need we check namespace which is newly added in yaml?

**Comment by hellosnow** [comment]
:shipit:

File: test/Microsoft.DocAsCode.Build.ManagedReference.WithPlugins.Tests/SplitClassPageToMemberLevelTest.cs (hunk 2 above)
Comment (vicancy) [inline test/Microsoft.DocAsCode.Build.ManagedReference.WithPlugins.Tests/SplitClassPageToMemberLevelTest.cs]: namespace added in yaml is ignored, it is in an invalid format and adding it to yaml to check if the test case fails

**Pull request merged**